recipes-bsp/u-boot/u-boot-beaglev_v2021.04.bb

```diff
@@ -0,0 +1,10 @@
+require recipes-bsp/u-boot/u-boot-common.inc
+require recipes-bsp/u-boot/u-boot.inc
+
+SRC_URI = "git://github.com/starfive-tech/u-boot.git;protocol=git;branch=Fedora"
+
+SRC_URI += " \
+    file://0001-include-configs-starfive-vic7100-adjust-fdt_addr_r.patch \
```

Comment:
this patch is causing the rootfs on SD card to be not found with fitImage and boot gets stuck at 
```
[    2.011165] mmc_host mmc0: Bus speed (slot 0) = 100000000Hz (slot req 400000Hz, actual 400000HZ div = 125)                                                                                                    
[    2.051162] mmc_host mmc1: Bus speed (slot 0) = 100000000Hz (slot req 400000Hz, actual 400000HZ div = 125)                                                                                                    
[    2.311113] dw_mmc 10010000.sdio1: Unexpected interrupt latency                                                                                                                                               
[    4.871335] Waiting for root device /dev/mmcblk0p2...                                                                                                                                                         
```

I will drop this patch in followup